Decred Wallet/Extensions/UIStackView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIStackView.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+extension UIStackView {
+    // Embed the UIstackiew components in a parent UIview with rounded edges as UIstackview does disregards it's layer.cornerRadius directive
+    func cornerRadius(_ radius: CGFloat) {
+        // Check if the stackview already has a corner radius providing view embedded
+        if self.subviews[0].accessibilityIdentifier == "cornerRadius" {
+            return
```

Comment:
Should update the corner radius of this subview before returning.

Reply:
no. this is to check if a corner radius has already been set. if it is already set, we silently return rather than adding another corner radius. UIStackView does not have or adhere to corner radius directives. this is a hack that places a rounded view behind all of its subviews at the very beginning of its subviews index and we do not want to add that rounded view more than once.